web/web.go

```diff
@@ -1100,61 +872,6 @@ func tmplFuncs(consolesPath string, opts *Options) template_text.FuncMap {
 	}
 }
 
-func (h *Handler) getTemplate(name string) (string, error) {
-	var tmpl string
-
-	appendf := func(name string) error {
-		f, err := ui.Assets.Open(path.Join("/templates", name))
-		if err != nil {
-			return err
-		}
-		defer f.Close()
-		b, err := ioutil.ReadAll(f)
-		if err != nil {
-			return err
-		}
-		tmpl += string(b)
-		return nil
-	}
-
-	err := appendf("_base.html")
-	if err != nil {
-		return "", errors.Wrap(err, "error reading base template")
-	}
-	err = appendf(name)
-	if err != nil {
-		return "", errors.Wrapf(err, "error reading page template %s", name)
-	}
-
-	return tmpl, nil
-}
-
-func (h *Handler) executeTemplate(w http.ResponseWriter, name string, data interface{}) {
-	text, err := h.getTemplate(name)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-	}
-
-	tmpl := template.NewTemplateExpander(
-		h.context,
-		text,
-		name,
-		data,
-		h.now(),
-		template.QueryFunc(rules.EngineQueryFunc(h.queryEngine, h.storage)),
-		h.options.ExternalURL,
-		nil,
-	)
-	tmpl.Funcs(tmplFuncs(h.consolesPath(), h.options))
-
-	result, err := tmpl.ExpandHTML(nil)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
-	io.WriteString(w, result)
-}
-
 // AlertStatus bundles alerting rules and the mapping of alert states to row classes.
 type AlertStatus struct {
```

Comment:
Looks like `AlertStatus` and `AlertByStateCount` can be removed now too, as they are unused.